.gitignore

```diff
@@ -12,3 +12,7 @@ build_branch_*/
 visit.masonry.docker.src.tar
 build-mb-*
 _logs
+*.tar.gz
```

Comment:
I'll approve the merge if you remove these changes.

Reply:
@brugger1 : Done.